source/renderer/app/types/walletRestoreTypes.js

```diff
@@ -6,6 +6,8 @@ export type RestoreWalletStep =
   | 'configuration'
   | 'success';
 
+export type ImportWalletStep = 'walletImportFile' | 'walletSelectImport';
```

Comment:
@DeeJayElly please update type definitions so that they start with a capital letter!

Reply:
Updated